addons/project/models/project.py

```diff
@@ -1676,6 +1678,17 @@ def _compute_stage_display(self):
         for task in self:
             task.stage_display = task.stage_id.name if task.project_id else task.personal_stage_id.stage_id.name
 
+    @api.depends('depend_on_ids', 'is_blocked')
+    def _compute_blocking_task_display(self):
+        for record in self:
+            if not record.is_blocked:
+                record.blocking_task_display = ""
+            else:
+                blocking_tasks = [task for task in record.depend_on_ids if not task.is_closed]
+                record.blocking_task_display = "Task " + blocking_tasks[0].name
+                record.blocking_task_display += " (and " + str(len(blocking_tasks)-1) + " other tasks)" if len(blocking_tasks) > 2 else ""
+                record.blocking_task_display += " (and one other task)" if len(blocking_tasks) == 2 else ""
```

Comment:
```suggestion
                record.blocking_task_display += " (and " + str(len(blocking_tasks)-1) + " other tasks)" if len(blocking_tasks) > 1 else ""
```
I think we should keep it simple. Also, will that be translated?

Reply:
You're right, I wasn't taking translation into account. I changed this method following the guidelines that I found here: https://www.odoo.com/documentation/16.0/developer/howtos/translations.html